app/models/chat_channel.rb

```diff
@@ -32,6 +32,21 @@ class ChatChannel < ActiveRecord::Base
         }
 
```

Comment:
We need to change the scopes for this model (`in_public_channel` and `in_dm_channel`) to point to the new column

Reply:
`type` is not replacement of `chatable_type`

Reply:
Yes sorry this is my bad I confused myself